cirq/contrib/qcircuit/qcircuit_diagram.py

```diff
@@ -47,7 +47,7 @@ def _render(diagram: circuits.TextDiagramDrawer) -> str:
         diagram2.write(2*w - 1, y, '&\\qw\\\\')
     grid = diagram2.render(horizontal_spacing=0, vertical_spacing=0)
 
-    output = '\Qcircuit @R=1em @C=0.75em {\n \\\\\n' + grid + '\n \\\\\n}'
+    output = '\\Qcircuit @R=1em @C=0.75em {\n \\\\\n' + grid + '\n \\\\\n}'
```

Comment:
Use an r-string.

Reply:
To produce the correct output I need the line feeds, so I can't make the entire string an r-string. I can use a combination however, like this: `output = r'\Qcircuit @R=1em @C=0.75em {' '\n' r' \\' '\n' + grid + '\n' r' \\' '\n}'`. What do you prefer?

Reply:
Ah, good point. If you can't use an r-string, use proper escaping with backashles intended for the output using `\\`. So it's good as is.